cypress/support/fragments/inventory/search/browseContributors.js

```diff
@@ -97,24 +96,23 @@ export default {
 
   checkNonExactSearchResult(contributorA, contributorZ) {
     cy.do([
-      row0.has({ content: '__A_test_contributor_would be here' }),
-      row1.has({ content: `${contributorA.name}${contributorA.contributorNameType}1` }),
-      MultiColumnListRow({ index: 2 }).has({ content:  `${contributorZ.name}${contributorZ.contributorNameType}1` }),
+      MultiColumnListRow({ index: 0 }).has({ content: '__A_test_contributor_would be here' }),
+      rowContributorName(contributorA.name, contributorA.contributorNameType).exists(),
+      rowContributorName(contributorZ.name, contributorZ.contributorNameType).exists(),
     ]);
   },
 
   checkExactSearchResult(contributorA, contributorZ) {
     cy.do([
       MultiColumnListCell(contributorA.name).has({ innerHTML: `<strong>${contributorA.name}</strong>` }),
-      row0.has({ content: `${contributorA.name}${contributorA.contributorNameType}1` }),
-      row1.has({ content:  `${contributorZ.name}${contributorZ.contributorNameType}1` }),
+      rowContributorName(contributorA.name, contributorA.contributorNameType).exists(),
```

Comment:
to test the expression that return true or false we usually use cy.expect(expression). Please update your function